src/main/java/view/EpisodeView.java

```diff
@@ -51,6 +56,7 @@ public EpisodeView(EpisodeViewModel viewModel) {
         JPanel buttonPanel = new JPanel(new FlowLayout(FlowLayout.CENTER));
         buttonPanel.add(playButton);
         buttonPanel.add(seeSummaryButton);
+        buttonPanel.add(backButton);
     }
 
     public void displayEpisode(Episode episode) {
```

Comment:
we are calling the TextChunk.toString() method here, but we never implemented that. It needs to be TextChunk.getText()